ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -96,7 +99,11 @@ class DrawerDemoController: DemoController {
         controller.resizingBehavior = resizingBehavior
         controller.adjustsHeightForKeyboard = adjustHeightForKeyboard
         controller.shouldRespectSafeAreaForWindowFullWidth = respectSafeAreaWidth
-
+        
+        if let maxDrawerHeight = maxDrawerHeight{
```

Comment:
do you have SwiftLint install? should have catch this kind of syntax error

Reply:
ran linter on all the modified files.